ros2node/ros2node/verb/list.py

```diff
@@ -37,4 +39,8 @@ def main(self, *, args):
         if args.count_nodes:
             print(len(node_names))
         elif node_names:
-            print(*sorted(n.full_name for n in node_names), sep='\n')
+            has_duplicates, sorted_names = annotate_duplicate_nodes(node_names)
```

Comment:
This seems like it would be annoying for people using this as a script. They're expecting a single complete node name per line in stdout, but now we are unconditionally adding something to some of the names. Maybe there needs to be an option to avoid this annotation, even if there are duplicate names. The warning to stderr is fine in any case.

Reply:
I hadn't thought of that use case - do you think the per-node annotation is valuable? Or, should we just leave it with the top level warning and call it a day? I could be convinced either way. Since the names are sorted anyways, probably there's not that much additional value added

Reply:
The warning only seems fine to me.